### PR TITLE
setup Version Target to 12.0

### PR DESCRIPTION
**Pull request by Yancerva**
setup Target Version to 12.0

### CODE REVIEW

File: Example/MTCircularSlider/ViewController.swift [hunk 1]
```diff
@@ -53,7 +53,7 @@ class ViewController: UIViewController {
 	}
 
 	fileprivate func setValueLabelText() {
-		valueLabel.text = String(Int(knobWithLabelView.value))
+		valueLabel.text = String(Int(knobWithLabelView.value))
```
Comment (EranBoudjnah) [inline Example/MTCircularSlider/ViewController.swift]: Why are we deleting the round caps from the example?

**Review by EranBoudjnah — COMMENTED** [review]
Looking good other than the line removed from the example 🙂

File: Example/MTCircularSlider/ViewController.swift (hunk 1 above)
Comment (Yancerva) [reply Example/MTCircularSlider/ViewController.swift]: I put that one, before didn't have that property, just to be sure for the rounds cap with the version 12.0 its working

or I leave that property active on that component?

Comment (EranBoudjnah) [reply Example/MTCircularSlider/ViewController.swift]: Let's revert the spacing change here and we're good to go 🙂

Comment (Yancerva) [reply Example/MTCircularSlider/ViewController.swift]: ok!

**Review by EranBoudjnah — APPROVED** [review]

**Pull request merged**